### ISSUE TITLE
Navbar Similarity Issue

### ISSUE THREAD

**Issue by bhavberi** [vedantmistryy/vedantmistryy.github.io#9]
Separate CSS Files for each page is giving styling errors for the header.

Solution: Try to have a single file containing CSS styling code for common elements - like header and footer

**Comment by vedantmistryy** [comment]
I know that, actually it was group project so my team mate created another file (copied the footer and header though)
You can give a try to merge all css in one file p.s create another branch please

**Comment by harshxtanwar** [comment]
hello @vedantmistryy @bhavberi I'd love to work on this issue, can I get assigned to it ?

**Comment by vedantmistryy** [comment]
@harshxtanwar Yeah sure you can contact me on discord, my username: vedantmistryy#5206

**Comment by harshxtanwar** [comment]
@vedantmistryy have just sent you a request on discord

**Comment by Satyabrat-Ojha** [comment]
I would love to contribute here and I have made a PR
Can you please assign it me or accept the contribution directly with label of Hacktoberfest?
